### PR TITLE
refactor(packages) Update spec 

### LINKED ISSUE #376: Deploy examples
# TL;DR

To gh-pages ?

## Backward compatibility

nothing

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Closes #376

### CODE REVIEW

File: packages/userdive/README.md [hunk 1]
```diff
@@ -23,13 +22,23 @@ npm install userdive --save
...
+const _ud = factory();
 
 _ud("create", "id", "auto");
 _ud("send", "pageview");
 ```
 
+```js
+import factory from "userdive";
+
+const myTracker = factory("myTracker"); // customize global name
```
Comment (Surgo) [inline packages/userdive/README.md]: `global object name` as same as agent.js doc

File: packages/linker/README.md [hunk 2]
```diff
@@ -2,16 +2,11 @@
...
 [![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
-[![codebeat badge](https://codebeat.co/badges/248f31a1-c73e-45e4-b1e0-a6154c1baaca)](https://codebeat.co/projects/github-com-userdive-agent-js-master)
-[![CodeFactor](https://www.codefactor.io/repository/github/userdive/agent.js/badge)](https://www.codefactor.io/repository/github/userdive/agent.js)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/007cedb2144843ebb45db871c04a0045)](https://app.codacy.com/app/USERDIVE/agent.js/dashboard)
-[![BCH compliance](https://bettercodehub.com/edge/badge/userdive/agent.js?branch=master)](https://bettercodehub.com/)
 
 [![Build Status](https://saucelabs.com/browser-matrix/userdive.svg)](https://saucelabs.com/open_sauce/user/userdive/builds)
 
-> TBA
+> CORS domain tracking using query parameter plugin
```
Comment (shoota) [inline packages/linker/README.md]: CORS is cross origin, including difference port. I think that should not use here.

File: examples/linker/linker.pug [hunk 3]
```diff
@@ -5,12 +5,11 @@ block footer_content
     !function(e,t,s,n,a,r,c,i){e[a]=e[a]||function(){(e[a].q=e[a].q||[]).push(arguments)},c=t.createElement(s),i=t.getElementsByTagName(s)[0],
     c.async=1,c.src=n,c.charset=r,c.setAttribute("data-ud-namespace",a),i.parentNode.insertBefore(c,i)}
     (window,document,"script","https://cdn.userdive.com/agent.js","_ud","UTF-8");
-    _ud("create", 'af57h6gb', 'auto');
+    _ud("create", 'af57h6gb', 'auto' { allowLinker: true });
     _ud("require", 'linker');
     _ud("send", "pageview");
-    _ud("linker", "autoLink", ['developers.userdive.com']);
-  script(src="https://cdn.userdive.com/linker.js")
+    _ud("linker:autoLink", ['developers.userdive.com']);
```
Comment (9renpoto) [inline examples/linker/linker.pug]: Connect to https://github.com/userdive/agent.js/issues/384

File: packages/linker/README.md [hunk 4]
```diff
@@ -2,16 +2,11 @@
...
 [![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
-[![codebeat badge](https://codebeat.co/badges/248f31a1-c73e-45e4-b1e0-a6154c1baaca)](https://codebeat.co/projects/github-com-userdive-agent-js-master)
-[![CodeFactor](https://www.codefactor.io/repository/github/userdive/agent.js/badge)](https://www.codefactor.io/repository/github/userdive/agent.js)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/007cedb2144843ebb45db871c04a0045)](https://app.codacy.com/app/USERDIVE/agent.js/dashboard)
-[![BCH compliance](https://bettercodehub.com/edge/badge/userdive/agent.js?branch=master)](https://bettercodehub.com/)
 
 [![Build Status](https://saucelabs.com/browser-matrix/userdive.svg)](https://saucelabs.com/open_sauce/user/userdive/builds)
 
-> TBA
+> Cross-domain tracking tracking using query parameter plugin
```
Comment (Surgo) [inline packages/linker/README.md]: `tracking` is duplicated

**Pull request merged**